gnomad/variant_qc/vqsr.py

```diff
@@ -1006,10 +1010,14 @@ def vqsr_workflow(
         b=b, utils=utils, gcp_billing_project=gcp_billing_project
     )
 
+    is_small_callset = False
+    is_large_callset = False
     if run_mode == "small":
         is_small_callset = True
     elif run_mode == "large":
         is_large_callset = True
+    else:
+        raise ValueError("Please specify a run mode: small or large.")
```

Comment:
shouldn't be necessary , lindo's code includes a default size of a cluster that is then modified if it is set as small or large. can probably be more clear about this in arguments though. should I include a 'standard' option in the list of options that is the default ?